L46-core-tls-credential-API.md

```diff
@@ -0,0 +1,339 @@
+Title
+----
+* Author(s): ZhenLian
+* Approver: markdroth, yashykt, jiangtaoli2016
+* Status: Draft
+* Implemented in:
+* Last updated: September 7, 2020
+
+## Abstract
+
+This proposal aims to enhance the security features provided by [this already implemented proposal](https://github.com/grpc/proposal/pull/98). 
```

Comment:
It looks like this proposal is designed to describe just the deltas relative to that earlier PR.  However, that earlier PR was never approved or merged, which is why the existing TLS creds API is still marked experimental.

I have closed the earlier PR in favor of this one.  This PR should be rewritten such that instead of referring to the earlier one, it completely replaces it.  In other words, it should describe the entire TLS creds API, not just the recent changes.

Reply:
+1 Agree with Mark. Let's describe the entire TLS creds API.

Reply:
Sure. I revised this PR. Could you please take a look again? Thanks!